specification/cognitiveservices/OpenAI.Inference/models/completions/completions.tsp

```diff
@@ -169,7 +170,7 @@ model Completions {
   """)
   @added(ServiceApiVersions.v2023_06_01_Preview)
   @projectedName("json", "prompt_filter_results")
-  promptFilterResults?: PromptFilterResult[];
+  promptFilterResults: PromptFilterResult[];
```

Comment:
Is `promptFilterResults` no longer optional? If yes, which implies Content Filtering result will in NonAzure OpenAI result now?

Reply:
Good catch! this should be optional.